tests/JustSaying.UnitTests/Messaging/Channels/SubscriberBusTests/WhenThereAreNoMessagesToProcess.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Amazon.SQS.Model;
+using JustSaying.AwsTools.MessageHandling;
+using JustSaying.TestingFramework;
+using NSubstitute;
+using Shouldly;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace JustSaying.UnitTests.Messaging.Channels.SubscriberBusTests
+{
+    public class WhenThereAreNoMessagesToProcess : BaseSubscriptionBusTests
```

Comment:
This class is testing something other than what it's name says it is testing.

Reply:
This one tests that when the queue successfully returns an empty array of messages, it continues to request more messages. Again it has that 3 in there 🤷